app/components/Auction.js

```diff
@@ -122,11 +126,15 @@ export default class Auction extends Component {
       bidIncrement,
       duration,
       startedAt,
-      isActive,
+      startBlock,
+      status,
       highestBid,
-      highestBidder
+      highestBidder,
+      cancelled,
     } = this.auction;
 
+    const isActive = status.equals(0);
```

Comment:
Is it better to do this mapping in the client or the `getAuction` function?

Reply:
To clarify, is it better to return `0` and store a mapping in the client that maps from `0 => active` OR return `status: "active"` from the `getAuction` function in the `AuctionBase` contract?

Reply:
@pfletcherhill I believe it's cheaper to do this on the client side instead of having the smart contract use `bytes32` for string values. I was saying to @mertcelebi that we might want to create a `AuctionBase.js` wrapper that the client uses to interact with the smart contract? This would be responsible for doing things like setting `status` to `active` if that's what we want.

Reply:
Yeah, totally. You could include that in the root store of the client, so you could do things like `this.props.store.auction.getAuction(...)`.

Reply:
Yeah, agree with @mykelp